src/Utilities/string_utils.h

```diff
@@ -139,11 +142,30 @@ inline bool string2bool(const std::string& s)
   }
   else
   {
-    APP_ABORT("string2bool received non-boolean string: " + s);
+    throw std::runtime_error("string2bool received non-boolean string: " + s);
     return false;
   }
 }
 
+template<class T>
+inline std::vector<T> convertStrToVec(const std::string& s)
```

Comment:
What format does this function expect the string to have?  Elements separated by spaces, commas, etc?

Reply:
it is through `istringstream>>` so by white space. I should add a comment. Should I rename the function to `convertStrToVecBySpaces`?

Reply:
No, the separator shouldn't be in the name. It should be documented, though.

Reply:
I will add a line of documentation in  #4168